atomicapp/constants.py

```diff
@@ -75,3 +75,6 @@
 PROVIDER_CONFIG_KEY = "providerconfig"
 PROVIDER_TLS_VERIFY_KEY = "providertlsverify"
 PROVIDER_CA_KEY = "providercafile"
+
+# Persistent Storage Formats
+PS_FORMAT = ["ReadWriteOnce", "ReadOnlyMany", "ReadWriteMany"]
```

Comment:
Would prefer a more meaningful name 'PERSISTENT_STORAGE_FORMAT' ? and is "format" the right word? By using the work "format" are we trying to make it generic?